packages/core/test/financial-templates/perpetual-multiparty/PerpetualPositionManager.js

```diff
@@ -770,7 +771,103 @@ contract("PerpetualPositionManager", function(accounts) {
     assert(await didContractThrow(positionManager.repay({ rawValue: "1" }, { from: sponsor })));
   });
 
-  it("Basic fees", async function() {
+  it("Basic funding rate fees", async function() {
```

Comment:
Pretty much copied (but a lot is stripped out that explicitly tests `_removeCollateral` and `_addCollateral`) from the regular fee test